[opencv4] Build failure

Package: opencv4[contrib,core,default-features,dnn,jpeg,ovis,png,python,quirc,tiff,webp]:arm64-osx -> 4.7.0#6

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 14.0.3.14030022
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: fba81a6a5 2023-07-14 (7 hours ago)

**To Reproduce**

`vcpkg x-set-installed --enforce-port-checks --allow-unsupported opencv[core,ovis,python]`

**Failure logs**

```
-- Using cached opencv-opencv-4.7.0-1.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/opencv-opencv-4.7.0-1.tar.gz
-- Applying patch 0001-disable-downloading.patch
-- Applying patch 0002-install-options.patch
-- Applying patch 0003-force-package-requirements.patch
-- Applying patch 0004-fix-eigen.patch
-- Applying patch 0005-fix-policy-CMP0057.patch
-- Applying patch 0006-fix-uwp.patch
-- Applying patch 0008-devendor-quirc.patch
-- Applying patch 0009-fix-protobuf.patch
-- Applying patch 0010-fix-uwp-tiff-imgcodecs.patch
-- Applying patch 0011-remove-python2.patch
-- Applying patch 0012-fix-zlib.patch
-- Applying patch 0015-fix-freetype.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean
-- Setting up python virtual environmnent...
-- Installing python packages: numpy
-- Setting up python virtual environmnent...finished.
-- Using cached opencv-cache/tiny_dnn/adb1c512e09ca2c7a6faef36f9c53e59-v1.0.0a3.tar.gz.
-- Using cached opencv-opencv_contrib-4.7.0.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/opencv-opencv_contrib-4.7.0.tar.gz
-- Applying patch 0007-fix-hdf5.patch
-- Applying patch 0016-fix-freetype-contrib.patch
-- Applying patch 0018-fix-depend-tesseract.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean
-- Using cached opencv-cache/wechat_qrcode/238e2b2d6f3c18d6c3a30de0c31e23cf-detect.caffemodel.
-- Using cached opencv-cache/wechat_qrcode/cbfcd60361a73beb8c583eea7e8e6664-sr.caffemodel.
-- Using cached opencv-cache/wechat_qrcode/6fb4976b32695f9f5c6305c19f12537d-detect.prototxt.
-- Using cached opencv-cache/wechat_qrcode/69db99927a70df953b471daaba03fbef-sr.prototxt.
-- Using cached opencv-cache/xfeatures2d/boostdesc/0ea90e7a8f3f7876d450e4149c97c74f-boostdesc_bgm.i.
-- Using cached opencv-cache/xfeatures2d/boostdesc/232c966b13651bd0e46a1497b0852191-boostdesc_bgm_bi.i.
-- Using cached opencv-cache/xfeatures2d/boostdesc/324426a24fa56ad9c5b8e3e0b3e5303e-boostdesc_bgm_hd.i.
-- Using cached opencv-cache/xfeatures2d/boostdesc/202e1b3e9fec871b04da31f7f016679f-boostdesc_binboost_064.i.
-- Using cached opencv-cache/xfeatures2d/boostdesc/98ea99d399965c03d555cef3ea502a0b-boostdesc_binboost_128.i.
-- Using cached opencv-cache/xfeatures2d/boostdesc/e6dcfa9f647779eb1ce446a8d759b6ea-boostdesc_binboost_256.i.
-- Using cached opencv-cache/xfeatures2d/boostdesc/0ae0675534aa318d9668f2a179c2a052-boostdesc_lbgm.i.
-- Using cached opencv-cache/xfeatures2d/vgg/e8d0dcd54d1bcfdc29203d011a797179-vgg_generated_48.i.
-- Using cached opencv-cache/xfeatures2d/vgg/7126a5d9a8884ebca5aea5d63d677225-vgg_generated_64.i.
-- Using cached opencv-cache/xfeatures2d/vgg/7cd47228edec52b6d82f46511af325c5-vgg_generated_80.i.
-- Using cached opencv-cache/xfeatures2d/vgg/151805e03568c9f490a5e3a872777b75-vgg_generated_120.i.
-- Using cached opencv-cache/data/7505c44ca4eb54b4ab1e4777cb96ac05-face_landmark_model.dat.
-- Configuring arm64-osx
-- Building arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/install-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/opencv4/portfile.cmake:451 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/install-arm64-osx-dbg-out.log</summary>

```
[1/1010] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCAROTENE_NS=carotene_o4t -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/3rdparty/carotene/hal/../include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/3rdparty/carotene/hal -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/core/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-unused-function -funwind-tables -fwrapv -g  -O0 -DDEBUG -D_DEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -std=c++11 -MD -MT 3rdparty/carotene/hal/CMakeFiles/tegra_hal.dir/dummy.cpp.o -MF 3rdparty/carotene/hal/CMakeFiles/tegra_hal.dir/dummy.cpp.o.d -o 3rdparty/carotene/hal/CMakeFiles/tegra_hal.dir/dummy.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/3rdparty/carotene/hal/dummy.cpp
[2/1010] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCAROTENE_NS=carotene_o4t -DWITH_NEON -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/3rdparty/carotene/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-unused-function -funwind-tables -fwrapv -Wno-unused-function -g  -O0 -DDEBUG -D_DEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -std=c++11 -MD -MT 3rdparty/carotene/hal/carotene/CMakeFiles/carotene_objs.dir/src/bitwise.cpp.o -MF 3rdparty/carotene/hal/carotene/CMakeFiles/carotene_objs.dir/src/bitwise.cpp.o.d -o 3rdparty/carotene/hal/carotene/CMakeFiles/carotene_objs.dir/src/bitwise.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/3rdparty/carotene/src/bitwise.cpp
...
Skipped 1496 lines
...
/stitching/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/superres/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/tracking/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/stereo/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/python/src2 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/python_bindings_generator -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -isystem /opt/homebrew/Frameworks/Python.framework/Versions/3.11/include/python3.11 -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-venv/lib/python3.11/site-packages/numpy/core/include -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-unused-function -Wno-deprecated-declarations -Wno-overloaded-virtual -Wno-unused-private-field -Wno-undef -g  -O0 -DDEBUG -D_DEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -std=c++11 -MD -MT modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_convert.cpp.o -MF modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_convert.cpp.o.d -o modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_convert.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/python/src2/cv2_convert.cpp
[1006/1010] : && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake -E rm -f lib/libopencv_superres4d.a && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ar qc lib/libopencv_superres4d.a  modules/superres/CMakeFiles/opencv_superres.dir/src/btv_l1.cpp.o modules/superres/CMakeFiles/opencv_superres.dir/src/btv_l1_cuda.cpp.o modules/superres/CMakeFiles/opencv_superres.dir/src/frame_source.cpp.o modules/superres/CMakeFiles/opencv_superres.dir/src/input_array_utility.cpp.o modules/superres/CMakeFiles/opencv_superres.dir/src/optical_flow.cpp.o modules/superres/CMakeFiles/opencv_superres.dir/src/super_resolution.cpp.o modules/superres/CMakeFiles/opencv_superres.dir/opencl_kernels_superres.cpp.o && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib lib/libopencv_superres4d.a && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake -E touch lib/libopencv_superres4d.a && :
[1007/1010] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -D_USE_MATH_DEFINES -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -Dopencv_python3_EXPORTS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/python/python3/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/python3 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/flann/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/hdf/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/imgproc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/intensity_transform/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/ml/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/phase_unwrapping/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/photo/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/plot/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/quality/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/reg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/surface_matching/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/xphoto/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/dnn/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/dnn_superres/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/features2d/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/fuzzy/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/hfs/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/img_hash/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/imgcodecs/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/line_descriptor/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/saliency/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/text/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/videoio/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/barcode/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/calib3d/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/datasets/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/highgui/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/mcc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/objdetect/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/ovis/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/rapid/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/shape/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/structured_light/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/video/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/videostab/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/wechat_qrcode/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/xfeatures2d/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/ximgproc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/xobjdetect/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/aruco/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/bgsegm/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/bioinspired/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/ccalib/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/dpm/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/face/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/optflow/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/stitching/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/superres/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/tracking/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/stereo/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/python/src2 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/python_bindings_generator -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -isystem /opt/homebrew/Frameworks/Python.framework/Versions/3.11/include/python3.11 -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-venv/lib/python3.11/site-packages/numpy/core/include -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-unused-function -Wno-deprecated-declarations -Wno-overloaded-virtual -Wno-unused-private-field -Wno-undef -g  -O0 -DDEBUG -D_DEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -std=c++11 -MD -MT modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_highgui.cpp.o -MF modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_highgui.cpp.o.d -o modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_highgui.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/python/src2/cv2_highgui.cpp
[1008/1010] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -D_USE_MATH_DEFINES -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -Dopencv_python3_EXPORTS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/python/python3/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/python3 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/core/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/flann/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/hdf/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/imgproc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/intensity_transform/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/ml/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/phase_unwrapping/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/photo/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/plot/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/quality/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/reg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/surface_matching/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/xphoto/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/dnn/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/dnn_superres/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/features2d/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/fuzzy/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/hfs/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/img_hash/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/imgcodecs/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/line_descriptor/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/saliency/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/text/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/videoio/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/barcode/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/calib3d/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/datasets/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/highgui/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/mcc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/objdetect/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/ovis/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/rapid/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/shape/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/structured_light/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/video/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/videostab/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/wechat_qrcode/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/xfeatures2d/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/ximgproc/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/xobjdetect/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/aruco/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/bgsegm/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/bioinspired/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/ccalib/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/dpm/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/face/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/optflow/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/stitching/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/superres/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/tracking/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-20e285eb9b.clean/modules/stereo/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/python/src2 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg/modules/python_bindings_generator -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-dbg -isystem /opt/homebrew/Frameworks/Python.framework/Versions/3.11/include/python3.11 -isystem /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/arm64-osx-venv/lib/python3.11/site-packages/numpy/core/include -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-unused-function -Wno-deprecated-declarations -Wno-overloaded-virtual -Wno-unused-private-field -Wno-undef -g  -O0 -DDEBUG -D_DEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -std=c++11 -MD -MT modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2.cpp.o -MF modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2.cpp.o.d -o modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/opencv4/src/4.7.0-87379d1df6.clean/modules/python/src2/cv2.cpp
[1009/1010] : && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-unused-function -Wno-deprecated-declarations -Wno-overloaded-virtual -Wno-unused-private-field -Wno-undef -g  -O0 -DDEBUG -D_DEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -bundle -Wl,-headerpad_max_install_names -Wl,-dead_strip   -undefined dynamic_lookup -o lib/python3/cv2d.cpython-311-darwin.so modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2.cpp.o modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_util.cpp.o modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_numpy.cpp.o modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_convert.cpp.o modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_highgui.cpp.o -L/usr/local/lib -Wl,-rpath,/usr/local/lib  lib/libopencv_core4d.a  lib/libopencv_flann4d.a  lib/libopencv_hdf4d.a  lib/libopencv_imgproc4d.a  lib/libopencv_intensity_transform4d.a  lib/libopencv_ml4d.a  lib/libopencv_phase_unwrapping4d.a  lib/libopencv_photo4d.a  lib/libopencv_plot4d.a  lib/libopencv_quality4d.a  lib/libopencv_reg4d.a  lib/libopencv_surface_matching4d.a  lib/libopencv_xphoto4d.a  lib/libopencv_dnn4d.a  lib/libopencv_dnn_superres4d.a  lib/libopencv_features2d4d.a  lib/libopencv_fuzzy4d.a  lib/libopencv_hfs4d.a  lib/libopencv_img_hash4d.a  lib/libopencv_imgcodecs4d.a  lib/libopencv_line_descriptor4d.a  lib/libopencv_saliency4d.a  lib/libopencv_text4d.a  lib/libopencv_videoio4d.a  lib/libopencv_barcode4d.a  lib/libopencv_calib3d4d.a  lib/libopencv_datasets4d.a  lib/libopencv_highgui4d.a  lib/libopencv_mcc4d.a  lib/libopencv_objdetect4d.a  lib/libopencv_ovis4d.a  lib/libopencv_rapid4d.a  lib/libopencv_shape4d.a  lib/libopencv_structured_light4d.a  lib/libopencv_video4d.a  lib/libopencv_videostab4d.a  lib/libopencv_wechat_qrcode4d.a  lib/libopencv_xfeatures2d4d.a  lib/libopencv_ximgproc4d.a  lib/libopencv_xobjdetect4d.a  lib/libopencv_aruco4d.a  lib/libopencv_bgsegm4d.a  lib/libopencv_bioinspired4d.a  lib/libopencv_ccalib4d.a  lib/libopencv_dpm4d.a  lib/libopencv_face4d.a  lib/libopencv_optflow4d.a  lib/libopencv_stitching4d.a  lib/libopencv_superres4d.a  lib/libopencv_tracking4d.a  lib/libopencv_stereo4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libhdf5_debug.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libszip_debug.a  lib/libopencv_quality4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreBitesStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libPlugin_DotSceneStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libCodec_AssimpStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libassimpd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpugixml.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpoly2tri.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libminizip.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libkubazip.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libSDL2d.a  -liconv  -Wl,-framework,CoreVideo  -Wl,-framework,Cocoa  -Wl,-framework,IOKit  -Wl,-framework,ForceFeedback  -Wl,-framework,Carbon  -Wl,-framework,CoreAudio  -Wl,-framework,AudioToolbox  -Wl,-framework,AVFoundation  -Wl,-framework,Foundation  -Wl,-weak_framework,GameController  -Wl,-weak_framework,Metal  -Wl,-weak_framework,QuartzCore  -Wl,-weak_framework,CoreHaptics  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreMeshLodGeneratorStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreOverlayStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libimguid.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libfreetyped.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libbz2d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libbrotlidec-static.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libbrotlicommon-static.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgrePropertyStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreTerrainStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgrePagingStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreRTShaderSystemStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreVolumeStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libPlugin_BSPSceneManagerStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libPlugin_OctreeSceneManagerStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libPlugin_PCZSceneManagerStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libPlugin_ParticleFXStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libRenderSystem_GLStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libRenderSystem_GL3PlusStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreGLSupportStatic.a  -Xlinker -framework -Xlinker OpenGL  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libCodec_STBIStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libCodec_FreeImageStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libFreeImaged.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libopenjp2.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebpdemux.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebpmux.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebp.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libsharpyuv.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebpdecoder.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjxrglued.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjpegxrd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/manual-link/libraw.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjasperd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblcms2.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOpenEXR-3_1_d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libIlmThread-3_1_d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libIex-3_1_d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libImath-3_1_d.a  -lOgreGLSupportStatic  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreMainStatic.a  -framework CoreFoundation -framework Foundation  lib/libopencv_phase_unwrapping4d.a  lib/libopencv_photo4d.a  lib/libopencv_objdetect4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libquirc.a  lib/libopencv_xfeatures2d4d.a  lib/libopencv_shape4d.a  lib/libopencv_optflow4d.a  lib/libopencv_ximgproc4d.a  lib/libopencv_tracking4d.a  lib/libopencv_plot4d.a  lib/libopencv_datasets4d.a  lib/libopencv_text4d.a  lib/libopencv_ml4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libtesseract.a  -lpthread  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libarchive.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libbz2d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libxml2.a  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libiconv.tbd  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libcharset.tbd  -ldl  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblz4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblzma.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libzstd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libcurl-d.a  -framework SystemConfiguration  -framework Security  -framework CoreFoundation  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libleptonica.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libgif.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libopenjp2.a  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libm.tbd  lib/libopencv_highgui4d.a  lib/libopencv_videoio4d.a  lib/libopencv_imgcodecs4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebpdecoder.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebp.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebpdemux.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebpmux.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libsharpyuv.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjpeg.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libtiffd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblzma.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjpeg.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libtiffd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblzma.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpng16d.a  -lm  -framework AppKit  -framework Accelerate  -framework AVFoundation  -framework CoreGraphics  -framework CoreMedia  -framework CoreVideo  -framework QuartzCore  -framework Cocoa  lib/libopencv_video4d.a  lib/libopencv_dnn4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libprotobufd.a  lib/libopencv_calib3d4d.a  lib/libopencv_features2d4d.a  lib/libopencv_flann4d.a  lib/libopencv_imgproc4d.a  lib/libopencv_core4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a  -framework OpenCL  3rdparty/lib/libtegra_hald.a && :
FAILED: lib/python3/cv2d.cpython-311-darwin.so 
: && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC   -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wno-delete-non-virtual-dtor -Wno-unnamed-type-template-args -Wno-comment -fdiagnostics-show-option -Qunused-arguments -Wno-semicolon-before-method-body -ffunction-sections -fdata-sections    -fvisibility=hidden -fvisibility-inlines-hidden -Wno-unused-function -Wno-deprecated-declarations -Wno-overloaded-virtual -Wno-unused-private-field -Wno-undef -g  -O0 -DDEBUG -D_DEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -bundle -Wl,-headerpad_max_install_names -Wl,-dead_strip   -undefined dynamic_lookup -o lib/python3/cv2d.cpython-311-darwin.so modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2.cpp.o modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_util.cpp.o modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_numpy.cpp.o modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_convert.cpp.o modules/python3/CMakeFiles/opencv_python3.dir/__/src2/cv2_highgui.cpp.o -L/usr/local/lib -Wl,-rpath,/usr/local/lib  lib/libopencv_core4d.a  lib/libopencv_flann4d.a  lib/libopencv_hdf4d.a  lib/libopencv_imgproc4d.a  lib/libopencv_intensity_transform4d.a  lib/libopencv_ml4d.a  lib/libopencv_phase_unwrapping4d.a  lib/libopencv_photo4d.a  lib/libopencv_plot4d.a  lib/libopencv_quality4d.a  lib/libopencv_reg4d.a  lib/libopencv_surface_matching4d.a  lib/libopencv_xphoto4d.a  lib/libopencv_dnn4d.a  lib/libopencv_dnn_superres4d.a  lib/libopencv_features2d4d.a  lib/libopencv_fuzzy4d.a  lib/libopencv_hfs4d.a  lib/libopencv_img_hash4d.a  lib/libopencv_imgcodecs4d.a  lib/libopencv_line_descriptor4d.a  lib/libopencv_saliency4d.a  lib/libopencv_text4d.a  lib/libopencv_videoio4d.a  lib/libopencv_barcode4d.a  lib/libopencv_calib3d4d.a  lib/libopencv_datasets4d.a  lib/libopencv_highgui4d.a  lib/libopencv_mcc4d.a  lib/libopencv_objdetect4d.a  lib/libopencv_ovis4d.a  lib/libopencv_rapid4d.a  lib/libopencv_shape4d.a  lib/libopencv_structured_light4d.a  lib/libopencv_video4d.a  lib/libopencv_videostab4d.a  lib/libopencv_wechat_qrcode4d.a  lib/libopencv_xfeatures2d4d.a  lib/libopencv_ximgproc4d.a  lib/libopencv_xobjdetect4d.a  lib/libopencv_aruco4d.a  lib/libopencv_bgsegm4d.a  lib/libopencv_bioinspired4d.a  lib/libopencv_ccalib4d.a  lib/libopencv_dpm4d.a  lib/libopencv_face4d.a  lib/libopencv_optflow4d.a  lib/libopencv_stitching4d.a  lib/libopencv_superres4d.a  lib/libopencv_tracking4d.a  lib/libopencv_stereo4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libhdf5_debug.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libszip_debug.a  lib/libopencv_quality4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreBitesStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libPlugin_DotSceneStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libCodec_AssimpStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libassimpd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpugixml.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpoly2tri.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libminizip.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libkubazip.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libSDL2d.a  -liconv  -Wl,-framework,CoreVideo  -Wl,-framework,Cocoa  -Wl,-framework,IOKit  -Wl,-framework,ForceFeedback  -Wl,-framework,Carbon  -Wl,-framework,CoreAudio  -Wl,-framework,AudioToolbox  -Wl,-framework,AVFoundation  -Wl,-framework,Foundation  -Wl,-weak_framework,GameController  -Wl,-weak_framework,Metal  -Wl,-weak_framework,QuartzCore  -Wl,-weak_framework,CoreHaptics  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreMeshLodGeneratorStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreOverlayStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libimguid.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libfreetyped.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libbz2d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libbrotlidec-static.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libbrotlicommon-static.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgrePropertyStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreTerrainStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgrePagingStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreRTShaderSystemStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreVolumeStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libPlugin_BSPSceneManagerStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libPlugin_OctreeSceneManagerStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libPlugin_PCZSceneManagerStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libPlugin_ParticleFXStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libRenderSystem_GLStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libRenderSystem_GL3PlusStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreGLSupportStatic.a  -Xlinker -framework -Xlinker OpenGL  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libCodec_STBIStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/plugins/ogre/libCodec_FreeImageStatic.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libFreeImaged.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libopenjp2.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebpdemux.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebpmux.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebp.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libsharpyuv.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebpdecoder.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjxrglued.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjpegxrd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/manual-link/libraw.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjasperd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblcms2.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOpenEXR-3_1_d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libIlmThread-3_1_d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libIex-3_1_d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libImath-3_1_d.a  -lOgreGLSupportStatic  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libOgreMainStatic.a  -framework CoreFoundation -framework Foundation  lib/libopencv_phase_unwrapping4d.a  lib/libopencv_photo4d.a  lib/libopencv_objdetect4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libquirc.a  lib/libopencv_xfeatures2d4d.a  lib/libopencv_shape4d.a  lib/libopencv_optflow4d.a  lib/libopencv_ximgproc4d.a  lib/libopencv_tracking4d.a  lib/libopencv_plot4d.a  lib/libopencv_datasets4d.a  lib/libopencv_text4d.a  lib/libopencv_ml4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libtesseract.a  -lpthread  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libarchive.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libbz2d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libxml2.a  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libiconv.tbd  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libcharset.tbd  -ldl  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblz4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblzma.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libzstd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libcurl-d.a  -framework SystemConfiguration  -framework Security  -framework CoreFoundation  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libleptonica.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libgif.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libopenjp2.a  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libm.tbd  lib/libopencv_highgui4d.a  lib/libopencv_videoio4d.a  lib/libopencv_imgcodecs4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebpdecoder.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebp.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebpdemux.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebpmux.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libsharpyuv.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjpeg.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libtiffd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblzma.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjpeg.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libtiffd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblzma.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpng16d.a  -lm  -framework AppKit  -framework Accelerate  -framework AVFoundation  -framework CoreGraphics  -framework CoreMedia  -framework CoreVideo  -framework QuartzCore  -framework Cocoa  lib/libopencv_video4d.a  lib/libopencv_dnn4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libprotobufd.a  lib/libopencv_calib3d4d.a  lib/libopencv_features2d4d.a  lib/libopencv_flann4d.a  lib/libopencv_imgproc4d.a  lib/libopencv_core4d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a  -framework OpenCL  3rdparty/lib/libtegra_hald.a && :
ld: library not found for -lOgreGLSupportStatic
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "lichtsteuerung",
  "version-string": "2.0 $.0",
  "dependencies": [
    {
      "name": "realsense2",
      "default-features": false,
      "features": []
    }
  ],
  "$overrides": [
    {
      "name": "capstone",
      "version": "4.0.1-120373dc"
    }
  ],
  "$builtin-baseline": "338f99eed499b6df1628829e40695b396a70a1f7"
}

```
</details>

--- a/ports/opencv4/0019-fix-ogre-dependency.patch
+++ b/ports/opencv4/0019-fix-ogre-dependency.patch
@@ -1,0 +1,9 @@
+--- a/modules/ovis/CMakeLists.txt
++++ b/modules/ovis/CMakeLists.txt
+@@ -27,4 +27,6 @@ ocv_create_module()
+ ocv_add_samples(opencv_aruco)
+ 
+ ocv_warnings_disable(CMAKE_CXX_FLAGS -Wunused-parameter)
++# The target always has the name OgreGLSupport
++string(REPLACE "OgreGLSupportStatic" "OgreGLSupport" OGRE_LIBRARIES "${OGRE_LIBRARIES}")
+ ocv_target_link_libraries(${the_module} ${OGRE_LIBRARIES})

--- a/ports/opencv4/portfile.cmake
+++ b/ports/opencv4/portfile.cmake
@@ -71,6 +71,7 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
  "nonfree"   OPENCV_ENABLE_NONFREE
  "openexr"   WITH_OPENEXR
  "opengl"    WITH_OPENGL
+ "ovis"      CMAKE_REQUIRE_FIND_PACKAGE_OGRE
  "png"       WITH_PNG
  "quirc"     WITH_QUIRC
  "sfm"       BUILD_opencv_sfm
@@ -183,6 +184,7 @@ if("contrib" IN_LIST FEATURES)
       0007-fix-hdf5.patch
       0016-fix-freetype-contrib.patch
       0018-fix-depend-tesseract.patch
+      0019-fix-ogre-dependency.patch
   )
   set(BUILD_WITH_CONTRIB_FLAG "-DOPENCV_EXTRA_MODULES_PATH=${CONTRIB_SOURCE_PATH}/modules")
 

--- a/ports/opencv4/vcpkg.json
+++ b/ports/opencv4/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "opencv4",
   "version": "4.8.0",
-  "port-version": 4,
+  "port-version": 5,
   "description": "computer vision library",
   "homepage": "https://github.com/opencv/opencv",
   "license": "Apache-2.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6018,7 +6018,7 @@
     },
     "opencv4": {
       "baseline": "4.8.0",
-      "port-version": 4
+      "port-version": 5
     },
     "opendnp3": {
       "baseline": "3.1.1",

--- a/versions/o-/opencv4.json
+++ b/versions/o-/opencv4.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "6deec5bdb97d4d3a667aa10e31698ce54abdc9a7",
+      "version": "4.8.0",
+      "port-version": 5
+    },
+    {
       "git-tree": "0d0ef2f6aa4911ca1cf367093c0b6a89cea97fe3",
       "version": "4.8.0",
       "port-version": 4

